src/server/seeds/development/05_orders.js

```diff
@@ -0,0 +1,28 @@
+exports.seed = function (knex) {
+  return knex('orders')
+    .del()
+    .then(function () {
+      return knex('orders').insert([
+        {
+          user_id: 1,
+          created_at: '2021-09-12 15:00:00',
+        },
+        {
+          user_id: 5,
+          created_at: '2021-09-12 12:00:00',
+        },
+        {
+          user_id: 9,
+          created_at: '2021-09-12 18:00:00',
+        },
+        {
+          user_id: 1,
```

Comment:
Since there are 2 orders from user 1 maybe it would be nice to have one payed order and make date older. So we will see history later on user page.

Reply:
yes ok , i will update it. Thanks @Jul-S 👍